persistence-api/src/main/java/io/stargate/db/PagingPosition.java

```diff
@@ -59,6 +62,44 @@
         .collect(Collectors.toMap(e -> e.getKey().name(), Entry::getValue));
   }
 
+  @Value.Lazy
+  default TableName table() {
+    Set<TableName> tables =
+        currentRow().keySet().stream()
+            .map(
+                c ->
```

Comment:
minor nit: Maybe `column` instead of `c`. It makes it easier to see this is iterating through key columns of without having to look at the types returned by `currentRow()`.

Reply:
Thinking about this more. It's not a big deal at all. Ignore. :)

Reply:
no worries. I'll change :)

Reply:
In 9c9c6af2 I renamed the variable, as well as the method (to `tableName()` just in case we need to add a method return the related `Table` object).